File: linear_search/linear_search.py
def linear_search(arr, x):
    """
    Performs a linear search
    :param arr: Iterable of elements
    :param x: Element to search for
    :return: Index if element found else None
    """
    l = len(arr)
    for i in range(l):
        if arr[i] == x:
            return i
    return None


def main():
    arr = range(100)
    index = linear_search(arr, 55)
    if index is None:
        print('Element not found in array')
    else:
        print('Element found at index %s' % index)


if __name__ == '__main__':
    main()

File: linear_search/linear_search.js
function linear_search (arr, x) {
	/*
	Performs a linear search
    :param arr: List of elements to search from
    :param x: Element to search for
    :return: Index if element found else -1
	*/
	for (let it in arr)
		if (arr[it] == x)
			return it;
	return -1;
}

function main () {
	A = [];
	for (let i = 1; i<=100; i++)
		A.push(i);
	index = linear_search(A, 12);
	if (index == -1) {
		console.log("Element was not found");
	} else {
		console.log("Element was found at %dth index", index);
	}
}

main();